docs/cse.md

```diff
@@ -46,15 +59,15 @@ Curl version 7.88
 	yum remove curl libcurl-devel
 	```
 
-#### Steps to Upgrade the OpenSSL* Toolkit to Version 1.1.1t
+#### Steps to Upgrade the OpenSSL* Toolkit to Version 3.0.8
 
 1. Pull the tarball:
 	```
-	wget https://www.openssl.org/source/openssl-1.1.1t.tar.gz
+	wget https://www.openssl.org/source/openssl-3.0.8.tar.gz
 	```
 2. Unpack the tarball with:
 	```
-	tar -zxf openssl-1.1.1t.tar.gz && cd openssl-1.1.1t
+	tar -zxf openssl-3.0.8.tar.gz && cd openssl-3.0.8
 	```
 3. Issue the command:
```

Comment:
In the linux.md it is mentioned as "./config --libdir=/usr/local/lib" . Should it be reflected here ?